p2p/validation/MESSAGE_VALIDATION_CONSENSUS.MD

```diff
@@ -0,0 +1,455 @@
+# Message Validation on Consensus messages
+
+## Summary
+
+This is a specification for the networking layer's message validation done on *QBFT* messages and *Decided* messages (aggregation of a quorum of commit messages). It is done by utilizing LibP2P's [extended validators](https://github.com/libp2p/specs/blob/master/pubsub/gossipsub/gossipsub-v1.1.md#extended-validators).
```

Comment:
this doesn't include partial sig messages?

Reply:
Not yet, unfortunately
Still ned to be specified

Reply:
Could you mark that TBD somewhere?